app/views/RoomsListView/Header/index.tsx

```diff
@@ -1,27 +1,28 @@
 import React, { PureComponent } from 'react';
-import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 
 import { toggleServerDropdown, closeServerDropdown, setSearch } from '../../../actions/rooms';
 import { withTheme } from '../../../theme';
 import EventEmitter from '../../../utils/events';
-import { KEY_COMMAND, handleCommandOpenServerDropdown } from '../../../commands';
+import { KEY_COMMAND, handleCommandOpenServerDropdown, IKeyCommandEvent } from '../../../commands';
 import { isTablet } from '../../../utils/deviceInfo';
 import { events, logEvent } from '../../../utils/log';
 import Header from './Header';
+import { IApplicationState, IBaseScreen } from '../../../definitions';
+import { ChatsStackParamList } from '../../../stacks/types';
 
-class RoomsListHeaderView extends PureComponent {
-	static propTypes = {
-		showServerDropdown: PropTypes.bool,
-		showSearchHeader: PropTypes.bool,
-		serverName: PropTypes.string,
-		connecting: PropTypes.bool,
-		connected: PropTypes.bool,
-		isFetching: PropTypes.bool,
-		theme: PropTypes.string,
-		server: PropTypes.string
-	};
+interface IRoomsListHeaderViewProps extends IBaseScreen<ChatsStackParamList, 'RoomsListHeaderView'> {
```

Comment:
I'm not sure about this `extends IBaseScreen<ChatsStackParamList, 'RoomsListHeaderView'>` 
I only did that, to use `dispatch` prop from `IBaseScreen`.  
But maybe, a better way to do is using `dispatch` directly at `IRoomsListHeaderViewProps`. 
WDYT guys ?